components/dispatcher/internal/dispatcher/dispatcher.go

```diff
@@ -183,5 +188,39 @@ func (d *Dispatcher) commitOffsets(logger *zap.Logger, consumer kafkaconsumer.Co
 			d.offsetMessages[index][partition.Partition] = partition.Offset
 		}
 	}
-	d.lastOffsetCommit = time.Now()
+	d.lastOffsetCommit[index] = time.Now()
+}
+
+// Convert Kafka Message To A Cloud Event, Eventually We Should Consider Writing A Cloud Event SDK Codec For This
+func convertToCloudEvent(message *kafka.Message) cloudevents.Event {
+	event := cloudevents.NewEvent(cloudevents.VersionV03)
+	event.SetData(message.Value)
+	for _, header := range message.Headers {
+		h := header.Key
+		v := string(header.Value)
+		switch h {
+		case "ce_datacontenttype":
+			event.SetDataContentType(v)
```

Comment:
same comment regarding constants.  It's ok if we leave them this way - just asking ; )